phpcs.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ruleset name="drupal">
+    <description>PHP CodeSniffer configuration for Drupal coding standards in Open Social.</description>
+    <file>./modules/custom</file>
+    <file>./modules/social_features</file>
+    <file>./social.profile</file>
+    <file>./social.install</file>
+    <file>./themes/socialbase</file>
+    <file>./themes/socialblue</file>
+    <config name="ignore_errors_on_exit" value="0"/>
+    <config name="ignore_warnings_on_exit" value="0"/>
+    <config name="default_standard" value="Drupal"/>
+
+    <arg name="extensions" value="php,module,inc,install,test,profile,theme" />
+
+    <exclude-pattern>*/node_modules/*</exclude-pattern>
+    <exclude-pattern>*.css</exclude-pattern>
+    <exclude-pattern>*/assets/*</exclude-pattern>
+    <exclude-pattern>*/components/*</exclude-pattern>
+    <exclude-pattern>*social.info.yml</exclude-pattern>
```

Comment:
```suggestion
    <exclude-pattern>social.info.yml</exclude-pattern>
```

Shouldn't this be the specific file instead of a wildcard in front of it?

Reply:
Not sure, got this from our check-coding-standard scripts, we can revise those later same as the other excluded patterns.